addons/website_sale/static/src/xml/website_sale.editor.xml

```diff
@@ -60,4 +60,12 @@
     </div>
 </div>
 
+<div t-name="website_sale.dialog.removeRibbon">
+    <div class="form-group row">
+        <label for="ppg_input" class="col-lg-auto col-form-label">
+            Are you sure to delete this ribbon ? It will be removed on all products.
```

Comment:
```suggestion
            Are you sure you want to delete this ribbon? It will be removed on all products.
```

Reply:
Of course. But this PR is blocked for now.